src/main/java/com/synopsys/integration/detect/workflow/componentlocationanalysis/GenerateComponentLocationAnalysisOperation.java

```diff
@@ -94,25 +83,4 @@ private File serializeInputToJson(File saveInputFileDir, Input libInput) throws
             throw new DetectUserFriendlyException("Failed to create component location analysis output file", ex, ExitCodeType.FAILURE_UNKNOWN_ERROR);
         }
     }
```

Comment:
Before entering the Component Location operation, the input data should  be reduced to a Component Set. We should not have to deal with BDIOs or Rapid / Stateless scan results here. That is taken care of in the parent OperationRunner class itself where all the failure conditions are first ruled out with appropriate messages.